src/main/java/io/github/terra121/util/CardinalDirection.java

```diff
@@ -1,21 +1,23 @@
 package io.github.terra121.util;
 public enum CardinalDirection {
-	N (337.5, 22.5, "North"),
-	S (157.5, 202.5, "South"),
-	E (67.5, 112.5, "East"),
-	W (247.5, 292.5, "West"),
-	NW (292.5, 337.5, "Northwest"),
-	SW (202.5, 247.5, "Southwest"),
-	SE (112.5, 157.5, "Southeast"),
-	NE (22.5, 67.5, "Northeast"),
-	UNKNOWN (360.5, 10000000.0, "Unknown");
+	N (337.5, 22.5, "North", "terra121.cardinal_directions.north"),
```

Comment:
that's a lot of repeated string constants.... maybe consider having these be generated in the constructor (e.g. `"terra121.cardinal_directions." + this.name().toLowerCase()`)